deps/obs-scripting/obs-scripting-python-frontend.c

```diff
@@ -161,6 +161,27 @@ static PyObject *set_current_transition(PyObject *self, PyObject *args)
 	return python_none();
 }
 
+static PyObject *get_transition_duration(PyObject *self, PyObject *args)
+{
+
+	int duration = obs_frontend_get_transition_duration();
+	PyObject *ret = PyLong_FromVoidPtr(duration);
```

Comment:
While I'm not sure that this can cause an actual problem, wouldn't it be better as something like
```suggestion
	PyObject *ret = PyLong_FromLong(duration);
```
The potential problem I can imagine is because [PyLong_FromVoidPtr](https://github.com/python/cpython/blob/32bd68c839adb7b42af12366ab0892303115d1d1/Objects/longobject.c#L974) interprets the parameter as an unsigned integer. Since it seems that `duration` should always be non-negative it's likely to be a non-issue. But since `duration` is using a signed type it seems best to keep it that way going into python.

Reply:
This throws an access violation. First off since this doesn't exist in our PY_EXTERN definitions yet, I have to add it myself. However in doing so it returns an invalid address point 
![image](https://user-images.githubusercontent.com/12771982/102275669-e0cc2500-3f79-11eb-9a4a-9a865e208243.png)
an explanation of why this behaviour is occurring is unknown

Reply:
Finally got a build environment setup locally using debian bullseye.

So on my setup just changing the call to PyLong_FromLong works and gets rid of the pointer from integer without a cast compiler warning. Adding the lines `PY_EXTERN PyObject *(*Import_PyLong_FromLong)(long);` and `#define PyLong_FromLong Import_PyLong_FromLong` to appropriate locations in `obs-scripting-python-import.h` also continues working fine. But it's been quite a while since I've done any serious C coding and I'm not completely sure when `NO_REDEFS` is or is not defined or what the actual purpose is. Obviously they're important for your platform.

I also noticed in this process that duration is being defined as `const int *` in `set_trannsition_duration` for both python and lua. Changing that to just `const int` calms the compiler warnings in both cases.